src/storage/main.cpp

```diff
@@ -86,6 +263,26 @@ void Manager::rescan()
     }
 }
 
+bool Manager::setDriveLocationLED(std::string driveSN, bool assert)
+{
+    for (const auto& [chan, mcu] : bplMCUs)
+    {
+        if (mcu->setDriveLocationLED(driveSN, assert))
+        {
+            return true;
```

Comment:
Have you had any success setting the value during testing?
I receive the 'false' result, but the error code is 0 and error message says 'Success'.

Reply:
Yes, it used to work, but I broke something in EM when I try to cleanup changes to create PR. Will work on this issue today.